### PR TITLE
Create buffered-nixpkgs.yml

### PR DESCRIPTION
**Pull request by grahamc**
Creates DeterminateSystems/nixpkgs-weekly, which updates every Tuesday ("patch Tuesday" style.)

- [ ] I have listed the project's license above, and it allows distribution.
- [ ] The project has declined to publish on their own.
- [ ] The project doesn't do versioned releases, OR the project was added to the versioned release mirror step.

### CODE REVIEW

File: .github/workflows/buffered-nixpkgs.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+jobs:
+  rolling:
+    name: Buffered mirror of branch ${{ matrix.repo }}#${{ matrix.branch }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: DeterminateSystems/nixpkgs-weekly
+            repo: NixOS/nixpkgs
+            branch: nixos-unstable
```
Comment (colemickens) [inline .github/workflows/buffered-nixpkgs.yml:15]: `nixpkgs` in the default registry maps to `nixpkgs-unstable`: https://github.com/NixOS/flake-registry/blob/9c69f7bd2363e71fe5cd7f608113290c7614dcdd/flake-registry.json#L330

Does it make sense to do the same here?

Comment (grahamc) [reply .github/workflows/buffered-nixpkgs.yml:15]: That works, yeah, as long as we're not planning on folks using it for NixOS machines!

Comment (grahamc) [reply .github/workflows/buffered-nixpkgs.yml:15]:
```suggestion
            branch: nixpkgs-unstable
```

**Review by colemickens — APPROVED** [review]
I realize it would make this less trivial but it would be nice if this updated "at most" once a week, rather than only attempting to update once a week.

**Comment by grahamc** [comment]
@colemickens We could give `fast-acting` (our service to rapidly push changes to mirrors) a `slow-acting` mode to implement something like that!

**Pull request merged**